### PR TITLE
signup/enterprise updates

### PR DESCRIPTION
**Pull request by deliaconstantino**
# Summary
- adds missing `h1` to page
- updates "Access Plane" to `h3` to match codebase convention for subheadings
- adds green check `Icon` for each bullet point
- preview [here](https://next-git-delia-signup-form-gravitational.vercel.app/signup/enterprise/)
Screen Shot 2022-05-12 at 3 45 52 PM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202262884845377

### CODE REVIEW

File: pages/signup/enterprise.tsx [hunk 1]
```diff
@@ -17,48 +24,42 @@ const Enterprise = () => {
...
+          mb={[2, 3]}
+        >
+          Contact our sales team
+        </Box>
+        <StyledUl>
+          {benefits.map((benefit, idx) => (
+            <LI key={idx} text={benefit} />
+          ))}
+        </StyledUl>
+        <Flex display="block">
```
Comment (sandylcruz) [inline pages/signup/enterprise.tsx:52]: This can be `Box` if using `display: block`

**Review by sandylcruz — APPROVED** [review]

File: pages/signup/enterprise.tsx (hunk 1 above)
Comment (deliaconstantino) [reply pages/signup/enterprise.tsx]: Updated

File: pages/signup/enterprise.tsx [hunk 2]
```diff
@@ -3,62 +3,63 @@ import { css } from "components/system";
...
+import Icon from "components/Icon";
 import NextImage from "next/image";
 import hosted from "./assets/hosted.png";
 
+const benefits: string[] = [
+  "Speak with our team",
+  "Get pricing information",
+  "Schedule a demo",
+];
+
```
Comment (jeffgaynor) [inline pages/signup/enterprise.tsx:17]: I'm late on this, but you can remove:

```ts
const benefits: string[] = [
  "Speak with our team",
  "Get pricing information",
  "Schedule a demo",
];

const LI = ({ text }: LIProps) => {
  return (
    <li>
      <CheckIcon /> {text}
    </li>
  );
};

<StyledUl>
  {benefits.map((benefit, idx) => (
    <LI key={idx} text={benefit} />
  ))}
</StyledUl>
```
and just do:
```ts
<StyledUl>
  <li><CheckIcon /> Speak with our team</li>
  <li><CheckIcon /> Get pricing information</li>
  <li><CheckIcon /> Get pricing information</li>
</StyledUl>
```

Which is shorter / simpler / easier to understand?

Comment (deliaconstantino) [reply pages/signup/enterprise.tsx:17]: Good point, will update this.

**Pull request merged**